staging/src/k8s.io/sample-apiserver/pkg/admission/plugin/banflunder/admission.go

```diff
@@ -44,7 +45,12 @@ func (d *disallowFlunder) Admit(a admission.Attributes) error {
 		return nil
 	}
 
-	flunderName := a.GetName()
+	metaAccessor, err := meta.Accessor(a.GetObject())
```

Comment:
that's interesting... from what I understand `a.GetName()` gets a name directly from a `URL`. On update one have to provide an existing resource name (in the `URL`) along with the fields to be updated (in the `Body`).  How can one change a name on update in this case ?